### PR TITLE
feat(back): add a configurable list of extensions to skip preview

### PR DESCRIPTION
**Pull request by floreal15**
https://github.com/tracim/tracim/issues/6500

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: backend/development.ini.sample [hunk 1]
```diff
@@ -295,6 +295,11 @@ preview_cache_dir = %(basic_setup.preview_cache_dir)s
 ## endpoint to get any other preview dimensions than allowed_dims will
 ## return error
 ; preview.jpg.restricted_dims = False
+## Preview skiplist, prevent preview for the content with an extension in the list
+## All item should be separated by a coma.
+; preview.skiplist=
+# examples with multiples values:
+# ; preview.skiplist= .ifc, .iges
```
Comment (Skylsmoi) [inline backend/development.ini.sample:302]: no need for this ;

Comment (floreal15) [reply backend/development.ini.sample]: yes, it's done

**Review by Skylsmoi — APPROVED** [review]

**Comment by floreal15** [comment]
After thinking it trough it has been decided to only have one parameter for both image preview and pdf preview

**Comment by Skylsmoi** [comment]
test review: the pdf is still generated even with the extension skipped

**Review by Skylsmoi — APPROVED** [review]

**Comment by Skylsmoi** [comment]
tested on branch

**Pull request merged**